cognite/powerops/custom_modules/power_model_v1/data_models/views/function_inputs/ShopPartialBidMatrixCalculationInput.view.yaml

```diff
@@ -0,0 +1,55 @@
+space: "{{powerops_models}}"
+externalId: ShopPartialBidMatrixCalculationInput
+name: ShopPartialBidMatrixCalculationInput
+description: Input to the task dispatcher in the shop bid process
+filter:
+  and:
+    - hasData:
+        - type: container
+          space: "{{powerops_models}}"
+          externalId: FunctionData
+    - equals:
+        property:
+          - node
+          - space
+        value: "{{powerops_instance_space}}"
+    - equals:
+        property:
+          - node
+          - type
+        value:
+          externalId: ShopPartialBidMatrixCalculationInput
+          space: "{{powerops_type_space}}"
+implements:
+  - space: "{{powerops_models}}"
+    externalId: PartialBidMatrixCalculationInput
+    version: "{{version}}"
+    type: view
+version: "{{version}}"
+properties:
+  priceProduction:
```

Comment:
In the priceProduction object we also have the shopResult object. What was the reason the have shopResult in each entry of this array?